futures-test/src/lib.rs

```diff
@@ -1,10 +1,6 @@
 //! Utilities to make testing [`Future`s](futures_core::Future) easier
 
-#![feature(
-    async_await,
-    await_macro,
-    futures_api,
-)]
+#![feature(async_await, await_macro, futures_api_const_fn_ptr)]
```

Comment:
futures_api_const_fn_ptr

^this shouldn't be necessary. What error did you hit without this?

Reply:
This patch was created when rust-lang/rust#59739 was opened, so I may have missed that is unnecessary.